game_function.php

```diff
@@ -0,0 +1,24 @@
+
+<?php
+
+function printStats(array $result): string
+{
+
+    $game_data = '';
+
+    foreach ($result as $data) {
+        '<div class="game">';
+        $game_data .= '<h3>Game: ' . $data['name'] . '</h3>';
+        '<div class="info">';
+        $game_data .= '<span>Rating: ' . $data['rating'] . '/5' . '</span><br />';
+        $game_data .= '<span>game duration: ' . $data['playing_time'] . ' minutes' . '</span><br />';
+        $game_data .= '<span>minimum players: ' . $data['min_players'] . '</span><br />';
+        $game_data .= '<span>maximum players: ' . $data['max_players'] . '</span><br />';
```

Comment:
Unsure how I feel about the use of br tags here. Technically they're used for spacing out text, which this is, so it's not that bad, but you could use a block level element such as a div instead of spans to achieve the same spacing without the need for the extra elements (br tags)